dokka-integration-tests/settings.gradle.kts

```diff
@@ -8,17 +8,19 @@ rootProject.name = "dokka-integration-tests"
 
 pluginManagement {
     includeBuild("../build-logic")
+    includeBuild("../build-settings-logic")
 
     repositories {
-        gradlePluginPortal()
         mavenCentral()
+        gradlePluginPortal()
```

Comment:
Unrelated change: prefer Maven Central over Gradle Plugin Portal.

Maven Central is more stable than Gradle Plugin Portal (which I think still connects to JCenter, which is unstable).